src/app/map/components/GoogleMaps/index.tsx

```diff
@@ -148,6 +164,22 @@ export const GoogleMaps = ({ setMapLoaded }: GoogleMapsProps) => {
         </GoogleMap>
       </LoadScript>
 
+      {noEvents && !loading ? (
```

Comment:
the transition issues seems to be caused by the conditional logic for loading state here I guess?.

Reply:
Yeah but if I don't add the conditional logic, it loads 'no events' message before 'there is a event' message gets displayed.. But going from 'there is a event'  to 'three is no event'  looks okay. So need to figure out how to render pop up message property. I will play around with it.